sdk/src/wallet/account/operations/transaction/prepare_transaction.rs

```diff
@@ -38,8 +38,11 @@ impl Account {
             output.verify_storage_deposit(rent_structure, token_supply)?;
         }
 
-        // validate amounts
-        if !OUTPUT_COUNT_RANGE.contains(&(outputs.len() as u16)) {
+        let is_burn_present = options.as_ref().map(|options| options.burn.is_some()).unwrap_or(false);
+
+        // Validate amounts. Validation of amounts shouldn't be performed if [`Burn`] is present.
```

Comment:
I know it was already like that but the comment is a tiny bit confusing, we're not validating amounts but numbers of outputs

Reply:
Yes, it makes sensse, fixed